SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -718,6 +718,23 @@ - (void)resetSignalStorage
     [TSAttachmentStream deleteAttachments];
 }
 
+/**
+ *  The user must unlock the device once after reboot before the database encryption key can be accessed.
+ */
++ (void)verifyDBKeysAvailableBeforeBackgroundLaunch
+{
+    if (CurrentAppContext().isMainApp && CurrentAppContext().mainApplicationState != UIApplicationStateBackground) {
+        return;
+    }
+
+    if (![TSStorageManager isDatabasePasswordAccessible]) {
```

Comment:
Is it even possible for an extension to be running in the background? I'm not sure if this method makes sense in the context of the app extension.